feature-libs/order/core/store/actions/consignment-tracking-by-id.action.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { StateUtils } from '@spartacus/core';
+import { ConsignmentTracking } from '@spartacus/order/root';
+import { CONSIGNMENT_TRACKING_BY_ID_ENTITIES } from '../order-state';
+
+export const LOAD_CONSIGNMENT_TRACKING_BY_ID =
+  '[Order] Load Consignment Tracking By ID Data';
+export const LOAD_CONSIGNMENT_TRACKING_BY_ID_FAIL =
+  '[Order] Load  Consignment Tracking By ID Data Fail';
+export const LOAD_CONSIGNMENT_TRACKING_BY_ID_SUCCESS =
+  '[Order] Load Consignment Tracking By ID Data Success';
+
+export class LoadConsignmentTrackingByID extends StateUtils.EntityLoadAction {
+  readonly type = LOAD_CONSIGNMENT_TRACKING_BY_ID;
+  constructor(
+    public payload: {
+      orderCode: string;
+      consignmentCode: string;
+      userId: string;
+    }
+  ) {
+    super(
+      CONSIGNMENT_TRACKING_BY_ID_ENTITIES,
+      `${payload.orderCode},${payload.consignmentCode}`
```

Comment:
[NITPICK]
Can we extract this "contatenation logic" to a separate function? I can see it's repeated in other places, but the shared function could be used there instead.

Reply:
done